doc-surrealdb_versioned_docs/version-latest/cli/start.mdx

```diff
@@ -124,10 +124,59 @@ __BEFORE YOU START :__ Make sure you’ve [installed SurrealDB](/docs/surrealdb/
 
 ## Example usage
 
-To display the current command-line tool version, along with the platform and architecture, in a terminal run the surreal version.
+This example will show how to host a SurrealDB server with the `surreal start` command, and then access the Surreal DB server using the [`surreal sql` command](/docs/surrealdb/cli/sql).
+
+To start a SurrealDB server, run the `surreal start` command, using the options below. This example stores the database in memory, with a username and password, hosted at 127.0.0.1:8000 (the default location).
 
 ```bash
-surreal version
+surreal start memory --user my_username --pass my_password
+```
+
+The following should appear in your console window, with potential slight differences depending on your operating system and version of SurrealDB. The server is actively running, and should be left alone until you want to stop hosting the SurrealDB server.
+
+Note the message "Started web server on 127.0.0.1:8000", which indicates where the server is being hosted and must be accessed by clients. The location 127.0.0.1:8000 is the default, and can be manually changed by specifing the `--bind` option of the `surreal start` command.
+
+```text
+ .d8888b.                                             888 8888888b.  888888b.
+d88P  Y88b                                            888 888  'Y88b 888  '88b
+Y88b.                                                 888 888    888 888  .88P
+ 'Y888b.   888  888 888d888 888d888  .d88b.   8888b.  888 888    888 8888888K.
+    'Y88b. 888  888 888P'   888P'   d8P  Y8b     '88b 888 888    888 888  'Y88b
+      '888 888  888 888     888     88888888 .d888888 888 888    888 888    888
+Y88b  d88P Y88b 888 888     888     Y8b.     888  888 888 888  .d88P 888   d88P
+ 'Y8888P'   'Y88888 888     888      'Y8888  'Y888888 888 8888888P'  8888888P'
+
+
+2024-09-04T08:54:48.898733Z  INFO surreal::env: Running 2.0.0-beta.1 for windows on x86_64
+2024-09-04T08:54:48.899231Z  INFO surrealdb::core::kvs::tr: Starting kvs store in memory
+2024-09-04T08:54:48.899652Z  INFO surrealdb::core::kvs::tr: Started kvs store in memory
+2024-09-04T08:54:48.900734Z  INFO surrealdb::core::kvs::tr: Credentials were provided, and no root users were found. The root user 'my_username' will be created
+2024-09-04T08:54:48.928488Z  INFO surrealdb::net: Started web server on 127.0.0.1:8000
+```
```

Comment:
@matthewjnield we've intentionally left this out as the version is displayed and that can be out of date with the next beta for example 
```suggestion
```